### PR TITLE
CleanNetworks in tests should wait until all networks are deleted

### PR DESCRIPTION
**Pull request by mmirecki**

### CODE REVIEW

File: test/util/namespaces/namespaces.go [hunk 1]
```diff
@@ -112,7 +112,25 @@ func CleanNetworks(operatorNamespace string, cs *testclient.ClientSet) error {
...
+	return waitForSriovNetworkDeletion(operatorNamespace, cs, 15*time.Second)
+}
+
+func waitForSriovNetworkDeletion(operatorNamespace string, cs *testclient.ClientSet, timeout time.Duration) error {
+	return wait.PollImmediate(time.Second, timeout, func() (bool, error) {
+		networks := sriovv1.SriovNetworkList{}
+		err := cs.List(context.Background(),
+			&networks,
+			runtimeclient.InNamespace(operatorNamespace))
+		if err != nil {
```
Comment (fedepaol) [inline test/util/namespaces/namespaces.go:124]: Can you return true if errors.IsNotFound? I am not sure how List behaves if there are no entries.

Comment (mmirecki) [reply test/util/namespaces/namespaces.go:124]: List() just returns an empty .Items list if not results are found

Comment (fedepaol) [reply test/util/namespaces/namespaces.go:124]: Good to know!

**Pull request merged**